compiler-core/src/javascript/typescript.rs

```diff
@@ -655,12 +655,14 @@ impl<'a> TypeScriptGenerator<'a> {
         )
         .to_doc();
 
+        let has_arguments = !arguments.is_empty();
+
         docvec![
             "export function ",
             name_with_generics(function_name, type_parameters),
             "(",
             docvec![break_("", ""), join(arguments, break_(",", ", ")),].nest(INDENT),
-            break_(",", ""),
+            break_(if has_arguments { "," } else { "" }, ""),
```

Comment:
If there's no arguments then we want to replace this whole section with the literal `():` 🙏

Reply:
Sure thing, updated to now keep the definition all on one line when there are no arguments.

Note that this does mean it can now exceed 80 columns.